uint32[] joints
uint32[] positions
---
string[] results
---
uint32[] positions

